src/CharityVault.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.6;
+
+import {ERC20} from "solmate/erc20/ERC20.sol";
+import {Auth} from "solmate/auth/Auth.sol";
+import {SafeERC20} from "solmate/erc20/SafeERC20.sol";
+
+import {WETH} from "./external/WETH.sol";
+import {CErc20} from "./external/CErc20.sol";
+
+import "./tests/utils/DSTestPlus.sol";
```

Comment:
dont think u need this anymore hehe

Reply:
Oh I thought we decided the `CharityVault` will mint and burn another token the `fcvToken`. But yea current code is not updated. Working on CharityVault right now.

Reply:
wot? just talking about importing DSTestPlus